db/init_db.js

```diff
@@ -278,18 +226,61 @@ async function createInitialProducts() {
   }
 }
 
+async function createInitialOrders(){
+  console.log('Starting to create orders...');
+
+  const ordersToCreate = [
+    {userId: 2, status: 'created'}, {userId: 1, status: 'created'}, {userId: 3, status: 'created'}
+  ];
+
+  try {
+    const orders = await Promise.all(ordersToCreate.map(createOrder));
+    console.log('Orders created:');
+    console.log(orders);
+    console.log('Finished creating orders!');
+  } catch (error) {
+    console.error('Error creating orders!')
+    throw error;
+  };
+};
+
+async function createInitialOrderProducts(){
+  try {
+    await client.query(`
```

Comment:
Maybe use `addProductToOrder` instead of hardcoding the query, that way you can change `addProductToOrder` in the future, and it changes the way this seed file works.